### PR TITLE
[Stage] V1 & V2 API

### PR DESCRIPTION
**Pull request by Aloento**
Re-implementation of API reads for SD2 V1 and SD3 V2.

### CODE REVIEW

**Comment by sgmv** [comment]
Please add the description for this PR.

File: .github/workflows/Web.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: Static Web Apps
```
Comment (sgmv) [inline .github/workflows/Web.yaml:1]: Please remove stage for deploying to GP, for test environment you should have a docker image first. Also please add a linter stage and stage for tests.

**Comment by sgmv** [comment]
I can't find any tests for this PR, could you add them?

File: .env [hunk 2]
```diff
@@ -1,4 +1,10 @@
 SD_BACKEND_URL="http://localhost:9000"
 SD_BACKEND_API="/api/v1"
```
Comment (sgmv) [inline .env:2]: api/v1 is deprecated, please remove it from source code

**Comment by Aloento** [comment]
> I can't find any tests for this PR, could you add them?

What kind of test do you want?

**Pull request closed without merging**